Dangerfile

```diff
@@ -1,4 +1,37 @@
 # frozen_string_literal: true
 
+def release_branch?
+  danger.github.branch_for_base.start_with?('release/') || danger.github.branch_for_base.start_with?('hotfix/')
+end
+
+def main_branch?
+  danger.github.branch_for_base == 'trunk'
+end
+
+def wip_feature?
+  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
+  has_wip_title = github.pr_title.include?('WIP')
+
+  has_wip_label || has_wip_title
+end
```

Comment:
I've seen these used in multiple repos. 

A suggestion for the next iteration would be to provide them as utils. I'm not sure how to name them, though. `Dangermattic::release_branch?`

Reply:
Yep, that's my idea. In fact, I believe I already have some of this in a non-merged PR, but now with the perspective from the apps it is worth seeing where it should live. Perhaps this could go in a util plugin?

I've created a small task for it: https://github.com/orgs/Automattic/projects/775/views/1?pane=issue&itemId=43079807

Reply:
> now having a better perspective of the integration in the apps, it is worth better thinking where it should live

💯 Nothing beats shipping a product to learn about its rough edges and how to improve it